scss/_base_code.scss

```diff
@@ -25,10 +25,7 @@ $code-sidebar-width: calc(#{$sph-inner} + (4 * #{$digit-width})) !default;
   samp {
     background-color: $color-mid-x-light;
     box-decoration-break: clone;
-    box-shadow: 0 0 0 0.25rem $color-mid-x-light;
-    margin-left: 0.25rem;
-    margin-right: 0.25rem;
-    word-break: break-all;
+    padding: 0.25rem;
```

Comment:
@lyubomir-popov Do we have a var for such padding on inline element?

Reply:
$spv-inner--x-small

Reply:
we should add $sph-inner--x-small as it is horizontal in this case

Reply:
Thanks, just added